corehq/apps/geospatial/templates/geospatial/case_management_base.html

```diff
@@ -23,3 +23,4 @@
     {% registerurl 'location_search' domain %}
     {% registerurl 'reassign_cases' domain %}
 {% endblock %}
+{% include 'geospatial/partials/index_alert.html' %}
```

Comment:
Checking if this is required as this is already included in the `case_management.html` ?

Reply:
You are right, this isn't required here as well. Addressed in [a4c7bb5](https://github.com/dimagi/commcare-hq/pull/35126/commits/a4c7bb5d50e939b2b55322d2e4df44ce96a57bbc).